docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Customise_Site_Reports.md

```diff
@@ -11,6 +11,20 @@ the site reports by clicking *Reports* in the left hand side bar and selecting t
 
 ![](../../../_images/sitereport.png) 
 
+## Counts in `ReportAdmin`
```

Comment:
What does that have to do with the change log? Can you point me to the logic controlling this?

Reply:
It's being linked to in the changelog. This info comes directly from the readme (as mentioned in this orbs description which links to https://github.com/silverstripe/silverstripe-reports/pull/139 for reference)